lib/orogen/gen/templates.rb

```diff
@@ -9,10 +9,22 @@ class << self
 	    attr_reader :templates
 	end
 
-        # Returns the directory where Orogen's lib part sits (i.e. where
-        # autobuild.rb and autobuild/ are)
+        # Returns the directory where oroGen's lib part sits, i.e.
+        # base_dir is defined by the directory where orogen.rb resides
         def self.base_dir
```

Comment:
Generally, with the packaging everything under lib will go into lib/ruby/vendor_ruby/ (so there is no 'lib' directory, but the main reference is orogen.rb). 
When moving the templates under lib/ then this is not necessary for this particular case.
Also, if I can assume that there are no further references relative to and going outside of lib/ (e.g. by plugins) then this fix is obsolete.
The documentation should still change as suggested.